include/net/net_core.h

```diff
@@ -33,23 +33,31 @@ extern "C" {
 
 /* Network subsystem logging helpers */
 
-#if defined(NET_LOG_ENABLED)
-#if !defined(SYS_LOG_DOMAIN)
-#define SYS_LOG_DOMAIN "net"
-#endif /* !SYS_LOG_DOMAIN */
-
-#undef SYS_LOG_LEVEL
-#ifndef NET_SYS_LOG_LEVEL
-#define SYS_LOG_LEVEL CONFIG_SYS_LOG_NET_LEVEL
+#if !defined(LOG_LEVEL)
+#if defined(NET_LOG_LEVEL)
```

Comment:
Why we need NET_LOG_LEVEL? Could we just use per-module LOG_LEVEL?

Reply:
This was done like this so that networking header files can have network specific checks.
For example if LOG_LEVEL is used, it might turn on things in network header files even if networking is disabled.